Modules/Microsoft365DSC/Modules/M365DSCUtil.psm1

```diff
@@ -2755,6 +2761,12 @@ function Update-M365DSCDependencies
         $ValidateOnly
     )
 
+
+    $Global:MaximumFunctionCount = 16000
```

Comment:
Why are we doing this twice?

Reply:
@NikCharlebois Hi, this doesn't actually solve the issue please read the whole thread #3454 on this topic.

Reply:
as @ricmestre points out - it does not solve the issue. this was for testing purposes.